eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -248,13 +248,7 @@ stages:
                       echo "##vso[task.setvariable variable=NpmToken]$npmToken"
                       echo "##vso[task.setvariable variable=Registry]$registry"
                     displayName: Detecting package archive_${{artifact.name}}
-                  - template: /eng/pipelines/templates/steps/npm-release-task.yml
```

Comment:
I would just comment this out with a TODO so this doesn't get lost. Or change the condition to false.